### PR TITLE
Two more glm replaces and small clean up.

### PR DESCRIPTION
**Pull request by ShFil119**
I've found two more. ;)

### CODE REVIEW

File: rwengine/src/objects/GameObject.cpp [hunk 1]
```diff
@@ -23,12 +23,10 @@ void GameObject::setRotation(const glm::quat& orientation) {
 }
 
 float GameObject::getHeading() const {
-    auto hdg = glm::roll(getRotation());
-    return hdg / glm::pi<float>() * 180.f;
+    return glm::degrees(glm::roll(getRotation()));
```
Comment (JayFoxRox) [inline rwengine/src/objects/GameObject.cpp]: I think having a "hdg" (maybe rename to: `heading`) variable makes sense as it shows the intention / documents what goes on (extract heading -> convert to degrees). Without it, I'm somewhat lost reading that line.

Other instance in file too.

The use of `glm::degrees` looks better though :)

File: rwengine/src/objects/GameObject.cpp [hunk 2]
```diff
@@ -23,12 +23,11 @@ void GameObject::setRotation(const glm::quat& orientation) {
 }
 
 float GameObject::getHeading() const {
-    auto hdg = glm::roll(getRotation());
-    return hdg / glm::pi<float>() * 180.f;
+    auto headingInRadians = glm::roll(getRotation());
+    return glm::degrees(headingInRadians);
```
Comment (JayFoxRox) [inline rwengine/src/objects/GameObject.cpp]: nit: the idea was also to keep `setHeading` and `getHeading` in sync. If both use the same variable name for the same content, it's easier to guess what goes on. anyhow, I'm fine with the current solution, maybe just something to keep in mind for next time :)

**Comment by JayFoxRox** [comment]
Please always provide a reason *why* you close your PRs.
This makes it easier for maintainers to improve the review process / create an overall better atmosphere.

**Comment by ShFil119** [comment]
Sorry, I forgot. :(
Current solution is good enough, so I didn't see the sense of holding "trash pull".

**Pull request closed without merging**